dynamic_config/dynamic_config_manager_test.py

```diff
@@ -40,34 +41,75 @@ def config():
   yield config
 
 
-def test_initialize_clusters_randomly(config):
+def test_initialize_clusters_randomly(cluster_config):
   """Test that clusters are randomly selected in initial configuration."""
-  with mock.patch('random.choice', side_effect=toggle_add_cluster) as _:
-    cm = dynamic_config_manager.DynamicClusterConfigManager(config)
+  with mock.patch('random.choice', side_effect=toggle_bool) as _:
+    cluster_manager = dynamic_config_manager.DynamicClusterConfigManager(cluster_config)
     config = discovery_pb2.DiscoveryResponse()
-    config.ParseFromString(cm.serialize())
+    config.ParseFromString(cluster_manager.serialize())
     assert (len(config.resources) == 1)
 
 
-def test_can_mutate_clusters(config):
+def test_can_mutate_clusters(cluster_config):
   """Test that we can randomly mutate the configuration."""
-  with mock.patch('random.choice', side_effect=toggle_add_cluster) as _:
-    cm = dynamic_config_manager.DynamicClusterConfigManager(config)
+  with mock.patch('random.choice', side_effect=toggle_bool) as _:
+    cluster_manager = dynamic_config_manager.DynamicClusterConfigManager(cluster_config)
 
-  cm.mutate()
+  cluster_manager.mutate()
   config = discovery_pb2.DiscoveryResponse()
-  config.ParseFromString(cm.serialize())
+  config.ParseFromString(cluster_manager.serialize())
 
   # Due to randomness in mutate, we check valid mutation states after invoking.
-  if cm.getLastMutateActionForTesting(
-  ) == dynamic_config_manager.DynamicClusterConfigManager.Action.ADD:
+  if cluster_manager.getLastMutateActionForTesting(
+  ) == dynamic_config_manager.DynamicConfigManager.Action.ADD:
     assert (len(config.resources) == 1 or len(config.resources) == 2)
-  elif cm.getLastMutateActionForTesting(
-  ) == dynamic_config_manager.DynamicClusterConfigManager.Action.REMOVE:
+  elif cluster_manager.getLastMutateActionForTesting(
+  ) == dynamic_config_manager.DynamicConfigManager.Action.REMOVE:
     assert (len(config.resources) == 1 or len(config.resources) == 0)
   else:
     raise NotImplementedError('Action: {} is not implemented.'.format(
-        cm.getLastMutateActionForTesting()))
+        cluster_manager.getLastMutateActionForTesting()))
+
+
+@pytest.fixture()
+def endpoints_config():
+  """Yield the config."""
+  config = endpoints_config_manager_pb2.DynamicEndpointsConfigManagerSettings()
+  config.cluster.name = "foo"
+  _add_endpoint(config.cluster, '127.0.0.1', 1234)
+  _add_endpoint(config.cluster, '127.0.0.1', 4567)
+  _add_endpoint(config.cluster, '192.168.0.1', 1234)
+  _add_endpoint(config.cluster, '192.168.0.1', 4567)
+  yield config
+
+
+def test_initialize_endpoints_randomly(endpoints_config):
```

Comment:
I am trying to confirm that these test targets execute in CI, but am not able to locate them. Under which CI target are these executed?

Reply:
very good catch, moved the test under `test/dynamic_config` since most of the CI targets do `bazel test //test/...`